xml/http_cookies.xml

```diff
@@ -517,8 +534,10 @@
     <param pos="0" name="service.product" value="Sage X3 Syracuse Web Server"/>
   </fingerprint>
 
-  <fingerprint pattern="^(gx_session_id|JROUTE)=">
+  <fingerprint pattern="^(gx_session_id|JROUTE)=" flags="REG_ICASE">
```

Comment:
I looked across multiple datasets and in a couple places on the web and only found `GX_SESSION_ID`, not the lower case version.  We may wish to just change to that in the regex and remove the `REG_ICASE` flag.

Reply:
will do this, thx for the 👀